yarn.lock

```diff
@@ -181,6 +186,28 @@
     semver "^5.4.1"
     source-map "^0.5.0"
 
+"@babel/core@~7.12.3":
```

Comment:
Are we able to consolidate versions so that we don't install multiple versions of babel libraries?

Reply:
Please what do you mean? I'm not sure if I understood